Terminal.Gui/Input/Key.cs

```diff
@@ -972,17 +973,18 @@ out parsedInt
 
     #endregion
 
-    private static Rune _shortcutDelimiter = new ('+');
+    private static Rune _separator = new ('+');
 
     /// <summary>Sets or gets the shortcut delimiter separator. The default is "+".</summary>
```

Comment:
"Gets or sets the separator character used when parsing and printing Keys. E.g. <c>Ctrl+A</c>. The default is '+'".